### PR TITLE
Update even more Equipment Effect descriptions and add icons where applicable

### PR DESCRIPTION
**Pull request by ammalagonc**

### CODE REVIEW

File: packs/equipment-effects/effect-spiny-lodestone-weapon.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Spiny Lodestone - Weapon",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Spiny Lodestone]</p>\n<p>After you cast a spell by Activating the <em>spiny lodestone</em>, your Strikes with the weapon deal an additional 1d4 piercing damage until the end of your next turn.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Spiny Lodestone]</p>\n<p>After you cast a spell by Activating the <em>spiny lodestone</em>, your Strikes with the weapon deal an additional 1d4 piercing damage.</p>"
```
Comment (simonward) [inline packs/equipment-effects/effect-spiny-lodestone-weapon.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Spiny Lodestone]</p>\n<p>Your Strikes with the weapon deal an additional 1d4 piercing damage.</p>"
```

File: packs/equipment-effects/effect-spiderfoot-brew-greater.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Spiderfoot Brew (Greater)",
     "system": {
         "description": {
-            "value": "<p>When you drink a spiderfoot brew, tiny clinging hairs grow on your hands and feet, granting you a climb Speed of 25 feet and a +3 item bonus to athletics checks made to climb for 1 hour.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Spiderfoot Brew (Greater)]</p>\n<p>You gain a climb Speed of 25 feet and a +3 item bonus to athletics checks made to climb.</p>"
```
Comment (simonward) [inline packs/equipment-effects/effect-spiderfoot-brew-greater.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Spiderfoot Brew (Greater)]</p>\n<p>You gain a climb Speed of 25 feet and a +3 item bonus to athletics checks made to Climb.</p>"
```

File: packs/equipment-effects/effect-spiderfoot-brew-lesser.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Spiderfoot Brew (Lesser)",
     "system": {
         "description": {
-            "value": "<p>When you drink a spiderfoot brew, tiny clinging hairs grow on your hands and feet, granting you a climb Speed of 15 feet and a +1 item bonus to athletics checks made to climb for 1 minute.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Spiderfoot Brew (Lesser)]</p>\n<p>You gain a climb Speed of 15 feet and a +1 item bonus to athletics checks made to climb.</p>"
```
Comment (simonward) [inline packs/equipment-effects/effect-spiderfoot-brew-lesser.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Spiderfoot Brew (Lesser)]</p>\n<p>You gain a climb Speed of 15 feet and a +1 item bonus to athletics checks made to Climb.</p>"
```

File: packs/equipment-effects/effect-spiderfoot-brew-moderate.json [hunk 4]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Spiderfoot Brew (Moderate)",
     "system": {
         "description": {
-            "value": "<p>When you drink a spiderfoot brew, tiny clinging hairs grow on your hands and feet, granting you a climb Speed of 20 feet and a +2 item bonus to athletics checks made to climb for 10 minutes.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Spiderfoot Brew (Moderate)]</p>\n<p>You gain a climb Speed of 20 feet and a +2 item bonus to athletics checks made to climb.</p>"
```
Comment (simonward) [inline packs/equipment-effects/effect-spiderfoot-brew-moderate.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Spiderfoot Brew (Moderate)]</p>\n<p>You gain a climb Speed of 20 feet and a +2 item bonus to athletics checks made to Climb.</p>"
```

File: packs/equipment-effects/effect-spellguard-shield.json [hunk 5]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Spellguard Shield",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Spellguard Shield]</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Spellguard Shield]</p>\n<p>You gain a +2 circumstance bonus to saving throws against spells that target you (as well as to AC), and you can Shield Block spells that target you if you have that action.</p>"
```
Comment (simonward) [inline packs/equipment-effects/effect-spellguard-shield.json:7]: I think the ac bonus RE can be removed, as it's already granted by raising the shield. That said, this effect is now defunct as the item itself contains the necessary REs.

File: packs/equipment/tacticians-helm.json [hunk 6]
```diff
@@ -9,7 +9,7 @@
...
...cs that feeds off your minor victories. The helm grants you a +1 item bonus to Warfare Lore checks. Also, a jewel adorns the brow of the helmet. This jewel becomes charged each time you hit a creature with a @UUID[Compendium.pf2e.actionspf2e.Item.Reactive Strike]. A tactician's helm can hold up to 2 charges, and its charges reset to 0 when you invest it.</p>\n<p><strong>Activate</strong> <span class=\"action-glyph\">1</span> command</p>\n<p><strong>Requirements</strong> The helm's jewel is charged</p>\n<p><strong>Frequency</strong> once per hour</p><hr /><p><strong>Effect</strong> One charge in the helm's jewel expires, and you choose one of the following effects.</p><ul><li><strong>Charge!</strong> Stride twice.</li><li><strong>Move It!</strong> You gain a +2 status bonus to Acrobatics and Athletics checks until the end of this turn. @UUID[Compendium.pf2e.equipment-effects.Item.Effect: Tactician's Helm (Move It!)]</li><li><strong>Protect!</strong> If you're wielding a shield, Stride to a space adjacent to an ally, then Raise your Shield.</li><li><strong>Re-Arm!</strong> Interact up to three times. Each of these actions must be used to do something on Changing Equipment.</li></ul>"
```
Comment (simonward) [inline packs/equipment/tacticians-helm.json:12]: I guess we no longer have an equivalent journal entry? PC p. 268 lists equipment actions.

Comment (ammalagonc) [reply packs/equipment/tacticians-helm.json:12]: That would be correct - had to trim some fat out of the GM screen. This one was one of the casualties :|

File: packs/equipment-effects/effect-spellguard-shield.json (hunk 5 above)
Comment (ammalagonc) [reply packs/equipment-effects/effect-spellguard-shield.json:7]: I'll remove the effect from the item and remove the ac bonus.

Comment (ammalagonc) [reply packs/equipment-effects/effect-spellguard-shield.json:7]: Nevermind - the shield doesn't have the effect.

**Pull request merged**